slides/advriskmin/slides-advriskmin-regression-further-losses.tex

```diff
@@ -149,7 +147,7 @@
 
 \end{columns}
 
-The log-cosh approach to obtain a differentiable approximation of the $L1$ loss can also be extended to  differentiable quantile/pinball losses. For more details on the log-cosh loss, see  \citebutton{Saleh and Saleh, 2024}{https://arxiv.org/pdf/2208.04564}.
+The log-cosh approach to obtain a differentiable approximation of the $L1$ loss can also be extended to  differentiable quantile/pinball losses. For more details on the log-cosh loss, see  \citelink{saleh2022statistical}.
```

Comment:
There is the unwritten rule: if a topic/slide has a "general background literature" paper then we put the citelink in the slide title. Only when we want to give a reference for a particular statement we use the citelink within the slides. So in this case => but ref in slide title of all log-cosh slide titles and remove sentences like "For more details on the log-cosh loss, see..."